### PR TITLE
[ICDS Dashboard] Add ward fields to awc location

### PR DESCRIPTION
**Pull request by Rohit25negi**
Code change adds two `commcare-user` case fields to the awc_location table.  This is a part of Bihar API.
ticket: https://app.asana.com/0/871068296963156/1160919599791898
3rd commit is unrelated, But I thought I could use this opportunity to improve the code. It just makes this location aggregation independent of ES as it happens in other places.

**Note**: This is a very time-sensitive Work of Bihar API. It would be great if we could wrap the Pr review soon.

### CODE REVIEW

**Review by czue — APPROVED** [review]
don't have a lot of context but code all looks good

File: custom/icds_reports/migrations/0172_add_ward_columns.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('icds_reports', '0171_add_sdr_field_and_view'),
+    ]
+
+    operations = [
+        migrations.RunSQL('ALTER TABLE awc_location ADD COLUMN awc_ward_1 VARCHAR'),
```
Comment (calellowitz) [inline custom/icds_reports/migrations/0172_add_ward_columns.py]: these should be `TEXT` not `VARCHAR`

**Review by calellowitz — CHANGES_REQUESTED** [review]

File: custom/icds_reports/migrations/0172_add_ward_columns.py (hunk 1 above)
Comment (Rohit25negi) [reply custom/icds_reports/migrations/0172_add_ward_columns.py]: @calellowitz  updated the type, Does it looks good now?

**Review by calellowitz — APPROVED** [review]
You should probably fix the lint errors but looks good

**Pull request merged**